exports/markdown/google_place_visits.go

```diff
@@ -7,39 +7,27 @@ import (
 )
 
 func (p *markdown) googlePlaceVisits() error {
-	var data []struct {
-		Date     string
-		Location string
-	}
+	var visits []google.PlaceVisit
 
 	p.DB().
 		Model(google.PlaceVisit{}).
-		Distinct(
-			"STRFTIME('%Y-%m-%d', duration_start_timestamp_ms) date",
-			"location_name location",
-		).
-		Find(&data)
+		Find(&visits)
```

Comment:
is it ok to load the whole table all at once without even selecting the required fields?